docs/en/install-upgrade/breaking.asciidoc

```diff
@@ -67,7 +67,7 @@ the complete list, go to {ref}/breaking-changes.html[{es} breaking changes].
 ++++
 
 This list summarizes the most important breaking changes in {elastic-sec} {version}. For
-the complete list, go to {security-guide-all}/8.8/release-notes-header-8.8.0.html#breaking-changes-8.8.0[{elastic-sec} breaking changes].
+the complete list, go to {security-guide-all}/8.8/release-notes-header-8.8.1.html#breaking-changes-8.8.1[{elastic-sec} breaking changes].
```

Comment:
I don't think we'll be publishing an HTML page called `release-notes-header-8.8.1.html`. AFAIK, the base page names for release notes are only minor versioned, not patch versioned. The heading reference in the URL (the part after the `#`) will be patch-versioned though.
```suggestion
the complete list, go to {security-guide-all}/8.8/release-notes-header-8.8.0.html#breaking-changes-8.8.1[{elastic-sec} breaking changes].
```

Reply:
Oh yeah, good catch and you're correct. I just checked this against an 8.7.x release, and you're right that the first half of the URL adheres to the base page version/name.